sdk/managedserviceidentity/Azure.ResourceManager.ManagedServiceIdentities/src/Custom/FederatedIdentityCredentialData.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using Azure.Core;
+using Azure.ResourceManager.Models;
+
+namespace Azure.ResourceManager.ManagedServiceIdentities
+{
+    public partial class FederatedIdentityCredentialData
+    {
+        /// <summary> The URL of the issuer to be trusted. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string Issuer
+        {
+            get
+            {
+                return IssuerUri?.AbsoluteUri;
```

Comment:
we should use .ToString() here to support both relative uri and absolute.

Reply:
In our serialization code, we use `AbsoluteUri` everywhere. See [example](https://github.com/Azure/autorest.csharp/blob/6c7eba34a5423d6f95de43e71c0a6142b272c87a/samples/Azure.Management.Storage/Generated/Models/EncryptionScopeKeyVaultProperties.Serialization.cs#L22), and our [code](https://github.com/Azure/autorest.csharp/blob/6c7eba34a5423d6f95de43e71c0a6142b272c87a/src/AutoRest.CSharp/Common/Generation/Writers/JsonCodeWriterExtensions.cs#L162).

Reply:
Spoke with @tg-msft and apparently this is a known limitation.  They feel its safe because we aren't aware of any actual use case where azure has relative Uris.  So for now lets keep this as `AbsoluteUri` and if/when customers report the exception being thrown we can address it then.